tests/vmi_multus_test.go

```diff
@@ -131,25 +131,26 @@ var _ = Describe("Multus", func() {
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 
-		// Create identical ptp crds in two different namespaces
+		// Create a ptp crd
 		result = virtClient.RestClient().
 			Post().
 			RequestURI(fmt.Sprintf(postUrl, tests.NamespaceTestDefault, "ptp-conf")).
```

Comment:
Why do you leave this one without PTP?

Reply:
Do you mean the one without the tuning plugin, right?

You are right, I should remove that and change the tests that are using it to the ptp-conf-tuning one otherwise we'll risk to have 0xfe on them as well.